src/cljs/main/broadfcui/page/workspace/data/copy_data_entities.cljs

```diff
@@ -18,37 +22,112 @@
+       :on-done (fn [{:keys [success? get-parsed-response]}]
+                  (swap! state dissoc :copying?)
+                  (when success?
+                    ((:on-data-imported props) (:type props)))
+                  (react/call :show-import-result this (get-parsed-response false) selected))}))
+   :show-import-result
+   (fn [{:keys [this props]} parsed-response selected]
+     (let [formatted-response (postwalk
+                               (fn [x]
+                                 (cond
+                                   (= x "entityName") "ID"
+                                   (= x "entityType") "Type"
+                                   (= x "conflicts") "Conflicting linked entities"
+                                   :else x))
+                               parsed-response)
+           copied (formatted-response "entitiesCopied")
+           hard-conflicts (formatted-response "hardConflicts")
+           soft-conflicts (postwalk
+                           #(if (= (second %) []) nil %)
+                           (formatted-response "softConflicts"))
+           import-type (:type props)]
+       (comps/push-ok-cancel-modal
+        {:header (if (not-empty copied)
+                   "Import Successful"
+                   "Unable to Import")
+         :content [:div {:style {:maxWidth 600}}
+                   (when (not-empty copied)
+                     [:div {}
+                      [:p {} "The following " import-type "s were imported successfully."]
+                      [comps/Tree {:data copied}]])
+                   (when (not-empty hard-conflicts)
+                     [:div {}
+                      [:p {}
+                       "The import did not complete because the following " import-type
+                       "s have the same IDs as entities already in this workspace."]
+                      [comps/Tree {:highlight-ends? true
+                                   :data hard-conflicts}]])
+                   (when (not-empty soft-conflicts)
+                     [:div {}
+                      [:p {} "The import did not complete because some of the " import-type
+                       "s that you selected are linked to entities that already exist in the
+                       destination workspace."]
+                      [:p {} "The conflicting entities are highlighted below."]
+                      [comps/Tree {:highlight-ends? true
+                                   :data soft-conflicts}]
+                      [:p {} "You may link the " import-type "s that you import to the existing
+                      entities in the destination workspace, by clicking " [:strong {} "Re-link"] "."]
+                      [:p {} "Re-linking will not import the conflicting entities from, or change the "
+                       import-type "s in, the source workspace."
+                       (common/render-info-box
+                        {:position "top"
+                         :text (react/create-element
```

Comment:
We generally use `create-element` only when we need to supply refs and need those to be associated with our component. Since you're not supplying refs here, you should be able to omit this (and the info box component should support that).